Integrations/VulnDB/VulnDB.yml

```diff
@@ -0,0 +1,893 @@
+      description: Vulnerability description (only human readable)
+      type: string
+    - contextPath: VulnDB.Vulnerability.SolutionDate
+      description: Vulnerability solution date
+      type: date
+    - contextPath: VulnDB.Vulnerability.DiscoveryDate
+      description: Vulnerability discovery date
+      type: date
+    - contextPath: VulnDB.Vulnerability.ExploitPublishDate
+      description: Exploit publish date
+      type: date
+    - contextPath: VulnDB.CVE-ExtReferences.Value
+      description: CVE- is a constant string
+      type: string
+    - contextPath: VulnDB.CvssMetrics.Id
+      description: Cvss reference value
+      type: number
+    - contextPath: VulnDB.CvssMetrics.AccessVector
+      description: Cvss access vector
+      type: string
+    - contextPath: VulnDB.CvssMetrics.AccessComplexity
+      description: Cvss access complexity
+      type: string
+    - contextPath: VulnDB.CvssMetrics.Authentication
+      description: Cvss metric authentication
+      type: string
+    - contextPath: VulnDB.CvssMetrics.ConfidentialityImpact
+      description: Cvss confidentiality impact
+      type: string
+    - contextPath: VulnDB.cvssMetrics.IntegrityImpact
+      description: Cvss integrity impact
+      type: string
+    - contextPath: VulnDB.CvssMetrics.AvailabilityImpact
+      description: Cvss availability impact
+      type: string
+    - contextPath: VulnDB.CvssMetrics.GeneratedOn
+      description: Cvss Metric date
+      type: date
+    - contextPath: VulnDB.CvssMetrics.Score
+      description: Cvss score
+      type: number
+    - contextPath: VulnDB.Vendor.Id
+      description: Vendor id
+      type: number
+    - contextPath: VulnDB.Vendor.Name
+      description: Vendor name
+      type: string
+    - contextPath: VulnDB.Products.Id
+      description: Products id
+      type: number
+    - contextPath: VulnDB.Products.Name
+      description: Products name
+      type: string
+    - contextPath: VulnDB.Products.Versions.Id
+      description: Product version id
+      type: number
+    - contextPath: VulnDB.Products.Versions.Name
+      description: Product versions name
+      type: string
+    - contextPath: VulnDB.Classification.Longname
+      description: Classification long name
+      type: string
+    - contextPath: VulnDB.Classification.Description
+      description: Classification description (only human readable)
+      type: string
+  - arguments:
+    - default: false
+      description: specific vendor id
+      isArray: false
+      name: vendor_id
+      required: false
+      secret: false
+    - default: false
+      description: specific vendor name (only human readable)
+      isArray: false
+      name: vendor_name
+      required: false
+      secret: false
+    - default: false
+      defaultValue: '20'
+      description: Maximum number of entries returned from the query, to avoid slow
+        response.
+      isArray: false
+      name: max_size
```

Comment:
I think we should change this to `max_results_amount` or something like that, please consult with @kirbles19